diff --git a/node_modules/react-native-event-listeners/index.js b/node_modules/react-native-event-listeners/index.js
index fa7b412..7dac79c 100644
--- a/node_modules/react-native-event-listeners/index.js
+++ b/node_modules/react-native-event-listeners/index.js
@@ -1,12 +1,7 @@
-import type from 'type-detect'
+const type = require("type-detect")
 
 class EventRegister {
 
-  static _Listeners = {
-    count: 0,
-    refs: {},
-  }
-
   static addEventListener(eventName, callback) {
     if (
       type(eventName) === 'string' &&
@@ -70,4 +65,9 @@ class EventRegister {
 
 } 
 
-export { EventRegister }
+EventRegister._Listeners = {
+  count: 0,
+  refs: {},
+}
+
+exports.EventRegister = EventRegister;
