diff --git a/node_modules/react-native-web/dist/modules/createDOMProps/index.js b/node_modules/react-native-web/dist/modules/createDOMProps/index.js
index d3796b2..f4c5008 100644
--- a/node_modules/react-native-web/dist/modules/createDOMProps/index.js
+++ b/node_modules/react-native-web/dist/modules/createDOMProps/index.js
@@ -408,6 +408,10 @@ var createDOMProps = function createDOMProps(elementType, props) {
     domProps.className = className;
   }
 
+  if (props.className) {
+    domProps.className = props.className;
+  }
+
   if (style) {
     domProps.style = style;
   } // OTHER
diff --git a/node_modules/react-native-web/dist/modules/pick/index.js b/node_modules/react-native-web/dist/modules/pick/index.js
index 343cdb0..4531f7d 100644
--- a/node_modules/react-native-web/dist/modules/pick/index.js
+++ b/node_modules/react-native-web/dist/modules/pick/index.js
@@ -11,7 +11,7 @@ export default function pick(obj, list) {
 
   for (var key in obj) {
     if (obj.hasOwnProperty(key)) {
-      if (list[key] === true || // Temporary until ARIA is mapped to explicit props
+      if (list[key] === true || key === "className" || // Temporary until ARIA is mapped to explicit props
       key.indexOf('aria-') === 0) {
         nextObj[key] = obj[key];
       }
