diff --git a/node_modules/react-native-web/dist/modules/createDOMProps/index.js b/node_modules/react-native-web/dist/modules/createDOMProps/index.js
index d3796b2..d92b5c7 100644
--- a/node_modules/react-native-web/dist/modules/createDOMProps/index.js
+++ b/node_modules/react-native-web/dist/modules/createDOMProps/index.js
@@ -404,8 +404,14 @@ var createDOMProps = function createDOMProps(elementType, props) {
       className = _styleResolver$resolv.className,
       style = _styleResolver$resolv.style;
 
+  domProps.className = ""
+
   if (className != null && className !== '') {
-    domProps.className = className;
+    domProps.className += className;
+  }
+
+  if (props.className) {
+    domProps.className +=" "+ props.className
   }
 
   if (style) {
diff --git a/node_modules/react-native-web/dist/modules/pick/index.js b/node_modules/react-native-web/dist/modules/pick/index.js
index 343cdb0..4531f7d 100644
--- a/node_modules/react-native-web/dist/modules/pick/index.js
+++ b/node_modules/react-native-web/dist/modules/pick/index.js
@@ -11,7 +11,7 @@ export default function pick(obj, list) {
 
   for (var key in obj) {
     if (obj.hasOwnProperty(key)) {
-      if (list[key] === true || // Temporary until ARIA is mapped to explicit props
+      if (list[key] === true || key === "className" || // Temporary until ARIA is mapped to explicit props
       key.indexOf('aria-') === 0) {
         nextObj[key] = obj[key];
       }
